internal/fields/validate.go

```diff
@@ -1207,7 +1213,7 @@ func (v *Validator) parseSingleElementValue(key string, definition FieldDefiniti
 			if len(errs) == 0 {
 				return nil
 			}
-			return errs
+			return errs.Unique()
```

Comment:
This error variable comes from `validateMapElement` that returns a multierror.Error.

That's why here I updated all the functions calling this to return also multierror.Error (`parseSingleElementValue` and others)